src/Api/Controllers/AuthController.php

```diff
@@ -331,6 +331,8 @@ public function loginBySocial() : Response
             $appleUserInfo = $source->validateAppleUser($request['social_id']);
             $request['social_id'] = $appleUserInfo->sub;
             $request['email'] = $appleUserInfo->email;
+        } elseif (!$source->validation($request['email'], $request['social_id'])) {
```

Comment:
@FredPeal I would put this on the else and just let the function throw the execption